test/e2e/bootstrapper_test.go

```diff
@@ -0,0 +1,113 @@
+package e2e
+
+import (
+	"fmt"
+	"github.com/openshift/windows-machine-config-operator/pkg/bootstrapper"
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+	"testing"
+	"time"
+)
+
+var ignitionFilePath string
+var kubeletPath string
+var installDir string
+
+func init() {
+	pflag.StringVar(&ignitionFilePath, "ignition-file", "C:\\Windows\\Temp\\worker.ign", "ign file location")
```

Comment:
Call out how you want to run the tests? Update the Makefile with test command.

Reply:
I think you can create those files if those flags are passed as empty values. As a side note, do not assume things in tests :)

Reply:
I too would like to see documentation on how the tests can be run. Please update the README with that information also.